### ISSUE TITLE
Mac_android android_semantics_integration_test failing to find AndroidSemanticsAction.accessibilityFocus action

### ISSUE THREAD

**Issue by jmagman** [flutter/flutter#94175]
Both tests started consistently failing in the devicelab a few hours ago:
https://chromium-swarm.appspot.com/task?id=576bafbbd738c910&o=true&w=true
https://chromium-swarm.appspot.com/task?id=576bafc358349410&o=true&w=true

Both are marked flaky, so the tree is not closed.

**Comment by godofredoc** [comment]
This is my fault, all the mac staging configs were updated to use mac 12.0.1 but the mac_android machine was not updated.

**Comment by jmagman** [comment]
Will mark these tests as unflaky since they have been fixed, which will move them back into the prod pool.

**Comment by mdebbar** [comment]
Just FYI: the `Mac_android run_release_test` has been highly flaky.

![image](https://user-images.githubusercontent.com/1278212/143904667-c9326210-5eeb-4d8d-a4eb-72cf7cb8748b.png)

**Comment by CaseyHillers** [comment]
@mdebbar are these flakes? It seems like  a real failure as it's pointing to `iOS Podfile.lock does not contain expected pods`

**Comment by mdebbar** [comment]
> @mdebbar are these flakes? It seems like a real failure as it's pointing to `iOS Podfile.lock does not contain expected pods`

What I meant is they fail frequently, and pass upon retries. I don't know the actual cause of the failures.

**Comment by jmagman** [comment]
> @mdebbar are these flakes? It seems like a real failure as it's pointing to `iOS Podfile.lock does not contain expected pods`

@CaseyHillers  Are you sure you were looking at `run_release_test`?  That error looks like https://github.com/flutter/flutter/issues/94372 from another test, and would only happen on iOS.

The `run_release_test` failures look like:
```
Task failed: flutter run --release had output on standard error.
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8829305881020289153/+/u/run_run_release_test/test_stdout

**Comment by jmagman** [comment]
Reopened https://github.com/flutter/flutter/issues/91625 to track the actual test flake.

**Comment by godofredoc** [comment]
\cc @gspencergoog

**Comment by godofredoc** [comment]
I can reproduce the flakiness locally with the latest master version of flutter/flutter and `TalkBack version is 9.0.0+316804458`.

```
11561 [android_semantics_integration_test] [STDOUT] stdout: [+1837 ms] 00:22 ^[[32m+7^[[0m: AccessibilityBridge Popup Controls Dropdown Menu has correct Android semantics^[[0m
11562 [android_semantics_integration_test] [STDOUT] stdout: [+3109 ms] 00:25 ^[[32m+7^[[0m^[[31m -1^[[0m: AccessibilityBridge Popup Controls Dropdown Menu has correct Android semantics ^[[1m^[[31m[E]^[[0m^[[0m
11563 [android_semantics_integration_test] [STDOUT] stdout: [        ]   Expected: AndroidSemanticsNode with className: android.view.View with actions: [AndroidSemanticsAction.accessibilityFocus, AndroidSemanticsAction.click] with flag isChecked: false with f      lag isEnabled: true with flag isFocusable: true
11564 [android_semantics_integration_test] [STDOUT] stdout: [        ]     Actual: AndroidSemanticsNode:<{flags: {isEditable: false, isEnabled: true, isDismissible: false, isCheckable: false, isChecked: false, isFocused: true, isLongClickable: false, isFocusa      ble: true, isPassword: false}, id: 73, text: null, rect: {height: 144, width: 330, bottom: 344.0, top: 296.0, left: 109.0, right: 219.0}, actions: [16, 128], className: android.view.View, liveRegion: 0, contentDescription: Item 1}>
11565 [android_semantics_integration_test] [STDOUT] stdout: [        ]      Which: Expected actions: [AndroidSemanticsAction.accessibilityFocus, AndroidSemanticsAction.click]
11566 [android_semantics_integration_test] [STDOUT] stdout: [        ]             Actual actions: [AndroidSemanticsAction.clearAccessibilityFocus, AndroidSemanticsAction.click]
11567 [android_semantics_integration_test] [STDOUT] stdout: [        ]             Unexpected: {AndroidSemanticsAction.clearAccessibilityFocus}
11568 [android_semantics_integration_test] [STDOUT] stdout: [        ]             Missing: {AndroidSemanticsAction.accessibilityFocus}
11569 [android_semantics_integration_test] [STDOUT] stdout: [        ]   Dropdown Item 1 doesn't have the right semantics
```
